src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -749,3 +749,10 @@ COOP_PINVOKE_HELPER(void, RhGetCurrentThreadStackBounds, (PTR_VOID * ppStackLow,
 {
     ThreadStore::GetCurrentThread()->GetStackBounds(ppStackLow, ppStackHigh);
 }
+
+ThreadExitCallback g_threadExitCallback;
```

Comment:
@sergiy-k I put comments in PalRedhawkCommon.h:

// Function to call when a thread is detached from the runtime
extern ThreadExitCallback g_threadExitCallback;

The exact purpose would depend on the class library.